src/core/radio-core.js

```diff
@@ -0,0 +1,32 @@
+const {knex} = require('../util/database').connect();
+import {deepChangeKeyCase} from '../util';
+import _ from 'lodash';
+
+
+function getStations(opts) {
+  return knex('radios').select('*').where(_getWhereClause(opts))
+    .then(rows =>
+      _.map(rows, row =>
+        deepChangeKeyCase(row, 'camelCase')
+      )
+    );
+}
+
+function _getWhereClause(filters) {
+  let whereClauses = {};
+
+  if (filters.cityId) whereClauses.city_id = filters.cityId;
+  if (filters.radioId) whereClauses.id = filters.radioId;
```

Comment:
We probably don't have an eslint rule to enforce curly braces in `if` statements but that has been the convention